apps/studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -70,7 +71,12 @@ export interface SupportFormProps {
 }
 
 const SupportForm = ({ setSentCategory, setSelectedProject }: SupportFormProps) => {
-  const { handleDocsSearchDebounced, searchState, searchState: state } = useDocsSearch()
+  const supabaseClient = useSupabaseClient()
```

Comment:
Injecting the client as a dependency instead of baking it into the hook because I want to minimize imports in `common` (where the hook is).

We didn't have the right package declarations for this before (but everything still worked fine because all the apps installed `@supabase/auth-helpers-react` for other reasons anyway). But now with the new tests it breaks testing if we don't fix it

Reply:
this is worrying me a bit as `@supabase/auth-helpers-react` is deprecated

Reply:
yeah all the apps are behind on this one still... 😢 we should probably have a plan to move off this (but not in this PR)